### PR TITLE
Allow composite key inside ELASTICSEARCH_UNIQ_KEY

### PR DESCRIPTION
**Pull request by aleroot**
Since I needed a composite key for my project, based on more than one item property. I've added composite key support to `ELASTICSEARCH_UNIQ_KEY`, it is enough to insert multiple columns separated by a space and you will have a composite key.

Example : 

`ELASTICSEARCH_UNIQ_KEY=field1 field2 field3`

### CODE REVIEW

File: scrapyelasticsearch/scrapyelasticsearch.py [hunk 1]
```diff
@@ -101,13 +101,14 @@ def process_unique_key(self, unique_key):
         return unique_key
 
     def get_id(self, item):
-        item_unique_key = item[self.settings['ELASTICSEARCH_UNIQ_KEY']]
-        unique_key = self.process_unique_key(item_unique_key)
```
Comment (jayzeng) [inline scrapyelasticsearch/scrapyelasticsearch.py]: what about passing in the key as a list? so the change looks like:

```
item_unique_key = item[self.settings['ELASTICSEARCH_UNIQ_KEY']]
if isinstance(item_unique_key, list):
    item_unique_key = '-'.join(item_unique_key)
```

Comment (aleroot) [reply scrapyelasticsearch/scrapyelasticsearch.py]: I think it is ok, if you prefer that way, you can change it that way. Do you plan to include the feature and update on pip repository ?

Comment (jayzeng) [reply scrapyelasticsearch/scrapyelasticsearch.py]: Yeah, I personally think it is cleaner that way. I will make the change and cut a release tonight.

**Comment by jayzeng** [comment]
@aleroot I made the change and released as 0.9.1 (https://pypi.python.org/pypi/ScrapyElasticSearch/0.9.1), let me know how it goes

**Pull request closed without merging**